### PR TITLE
Changed where Generic objects list gets rendered on Service details screen

### PR DESCRIPTION
**Pull request by chalettu**
@miq-bot add_label bug
@miq-bot add_label  gaprindashvili/yes

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1515945

### CODE REVIEW

**Comment by chalettu** [comment]
demo_go
Sample screenshot of fix.  This ensures that even if no VM's exist that we show Generic Objects on a service.

**Comment by chalettu** [comment]
@Loicavenel and @bascar .  Could you take a look at this and see if update looks as you are expecting it to.

**Comment by chriskacerguis** [comment]
@Loicavenel @bascar need approval on this one.  It does change the UX a bit, so @serenamarie125 could you please let us know if that is ok?

File: client/app/services/service-details/service-details.html [hunk 1]
```diff
@@ -347,24 +347,28 @@ <h2 translate>Resources</h2>
...
-                        </div>
-                    </li>
-                </ul>
+                </ul>      
             </section>
+         
         </div>
     </div>
-
+    <div ng-if="vm.genericObjects.length > 0">
```
Comment (AllenBW) [inline client/app/services/service-details/service-details.html]: `<div ng-if="vm.genericObjects.length">`
will be true when anything is greater than 0

File: client/app/services/service-details/service-details.html [hunk 2]
```diff
@@ -347,24 +347,28 @@ <h2 translate>Resources</h2>
...
-                        <div class="service-details-resource-list-container"
-                             ng-class="{'collapse': !genericObject.isExpanded}">
-                            <generic-objects-list generic-object="genericObject"
-                                                  on-update="vm.setGenericObjectViewState(object)"></generic-objects-list>
-                        </div>
-                    </li>
-                </ul>
+                </ul>      
             </section>
+
```
Comment (AllenBW) [inline client/app/services/service-details/service-details.html]: eeeekkkk if we don't need da space...

**Comment by AllenBW** [comment]
@chalettu might not be a bad idea to give this area a header kinda similar "Properties" and "Resources" ?  
screen shot 2017-11-16 at 1 53 46 pm

**Comment by chalettu** [comment]
Ok, updated the screenshot to put this under the Resources section.  It will show even in instances when no VM's exist for a service. 
updated_go_screen

**Comment by AllenBW** [comment]
@chalettu oh gosh you're gonna 🔪 me, from the most recent ss, looks like the horizontal bars under above and below resources went away? or maybe my 👁 deceives me...

**Comment by chalettu** [comment]
They are still there.  They are so light its hard to see.  If you view the video full screen you should see em.

**Comment by AllenBW** [comment]
Yeah viewed it full screen... seem em around relationships just not resources... 🤔

**Comment by chalettu** [comment]
lines
You can see them in this image.  They are a little lighter than those other line used by relationships, but that wasn't a change made in this PR.  I merely moved a few things around so that GO's show up in the instance no other resources are defined.

**Comment by chalettu** [comment]
@serenamarie125 , please disregard the earlier images.  The very last one is the one we implemented and that one does show things being grouped appropriately under the Resources Title.

**Comment by chriskacerguis** [comment]
@chalettu is this ready to go for @serenamarie125 to review?

**Comment by AllenBW** [comment]
@chalettu Any update on implementing the requested changes?

**Comment by chalettu** [comment]
@AllenBW , code is all set for this PR , I was working on getting onto an appliance so I do a screenshot demo of the requested addition.  I will double check with Loic to see if we have a appliance to use.

**Comment by chalettu** [comment]
updated_image
Screenshot showing the updated count header for Generic Object types.

**Comment by chalettu** [comment]
@serenamarie125 , can you take a quick look at the updated screen that includes the count of Genric objects?  Thanks in advance.

**Comment by serenamarie125** [comment]
Location of Generic Objects is 👍 and thanks for adding the count!

**Comment by serenamarie125** [comment]
@miq-bot add_label ux/approved

**Comment by serenamarie125** [comment]
@miq-bot remove_label ux/review

**Comment by simaishi** [comment]
Gaprindashvili backport details:

```
$ git log -1
commit 995ee638a89426fa08d4d18252813e9d7d0f3742
Author: Allen Wight <allen.b.wight@gmail.com>
Date:   Thu Dec 7 12:48:41 2017 -0500

    Merge pull request #1246 from chalettu/Generic-objects-list
    
    Changed where Generic objects list gets rendered on Service details screen
    (cherry picked from commit 32ca812a162cf49bbb43c72b0fe931af86bde3cb)
    
    https://bugzilla.redhat.com/show_bug.cgi?id=1524721
```

**Pull request merged**